src/components/ProfileForm.tsx

```diff
@@ -4,7 +4,7 @@ import { ProfileSelect } from './ProfileSelect';
 import { SelectOption } from '../models/SelectOption';
 
 export interface ProfileProps {
-  readonly optionsByProfile: Map<String, SelectOption[]>;
+  readonly optionsByProfile: Map<string, SelectOption[]>;
```

Comment:
is this actually a `Map`, or should it be a `Record`?
```suggestion
  readonly optionsByProfile: Record<string, SelectOption[]>;
```

Reply:
Currently, it's a `Map`